src/app/modules/shared_models/message_model.nim

```diff
@@ -186,11 +194,18 @@ QtObject:
     of ModelRole.MentionedUsersPks:
       result = newQVariant(item.mentionedUsersPks.join(" "))
 
+  proc updateItemAtIndex(self: Model, index: int) =
+    let ind = self.createIndex(index, 0, nil)
```

Comment:
Not knowing much (yet) about NIM, but in C++ you'd rather call:

`const auto ind = this.index(index /*row*/, 0 /*col*);`

In other words, I don't see the reason for creating a new index, it should rather call this method: https://doc.qt.io/qt-5/qabstractlistmodel.html#index

Reply:
Anyway, I consider it not to be a blocker of this PR. We should possibly substitute `createIndex` with `index` in follow-up PR.

Reply:
Sure, let's potentially fix that in some followup PR